.github/workflows/deploy-gcp-tests.yml

```diff
@@ -357,6 +310,72 @@ jobs:
           --mount type=volume,src=${{ inputs.disk_prefix }}-${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }},dst=${{ inputs.root_state_path }}/${{ inputs.lwd_state_dir }} \
           ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}"
 
+  create-state-image:
+    name: Create ${{ inputs.test_id }} cached state image
```

Comment:
Question: I'd like to block merging if this step fails. Is there an easy way to do that?

(I think we might have to add a patch entry for each test that calls this re-usable workflow.)

Reply:
We're using `needs: [ test-without-cached-state, test-with-cached-state ]` in this step now, so we shouldn't worry about this step but the ones this step depends on.

`gcloud compute images create` only fails if the instance does not exist, which won't be the case if we depend on the instance being created and running successfully.